lib/auth.ts

```diff
@@ -41,6 +43,7 @@ export const authorizedPage = (
 ): GetServerSideProps => {
   return async (context: Context) => {
     if (await verifyServerSideAuthorization(context)) {
+      context.res.setHeader("Cache-Control", "no-cache");
```

Comment:
These are safety valves so that our auth'd routes don't accidentally get cached by the CDN. I think they're fine to be cached by the browser (which is why I didn't add `no-store`).